.claude/plugins/@sea/sea-domain-forge/commands/artifact.md

```
@@ -0,0 +1,56 @@
+  checklists, kanban boards, decision trees, mind maps, and timelines. Artifacts are
+  semantically anchored to the Knowledge Graph and can be linked to CMMN cases.
+  Use: /sea-domain-forge:artifact [type] [context?]
+---
+
+# /sea-domain-forge:artifact
+
+Generate a CADSL cognitive artifact.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)